### ISSUE TITLE
Elementor Pro v3.2.3 - Increase file size of frontend.min.js

### ISSUE THREAD

**Issue by astbarc** [elementor/elementor#15278]
## Prerequisites

- [* ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [* ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ *] This bug happens with only Elementor plugin active (and Elementor Pro).
- [* ] This bug happens with a default WordPress theme active.
- [* ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

Version 3.3.0 of elementor pro increase the size of the file frontend.min.js in 40kb for the minified version and 146kb for the regular version.

¿What is the reason for this increase? I think this could affect speed and performance.

**Comment by shilo-ey** [comment]
@astbarc Thanks! nice catch!

We are working on a fix and it will be released soon

**Comment by astbarc** [comment]
Hi @shilo-ey, that's great

Thanks for your support.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.3.1

Feel free to update

Thanks!
